.eslintrc.js

```diff
@@ -63,7 +63,8 @@ module.exports = {
     'react/require-default-props': 'off',
     'react/sort-comp': 'off',
     'jsx-a11y/mouse-events-have-key-events': 'off',
-    'global-require': 'off'
+    'global-require': 'off',
+    'no-eval': 'off'
```

Comment:
will remove the global disable of the 'no-eval' rule and add the eslint comment at the single line involved.